modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -689,6 +689,27 @@ CV__DNN_INLINE_NS_BEGIN
         static Ptr<Layer> create(const LayerParams& params);
     };
 
+    class CV_EXPORTS QuantizeLinearLayer : public Layer
+    {
+    public:
+        static Ptr<QuantizeLinearLayer> create(const LayerParams& params);
+    };
+
+    class CV_EXPORTS DequantizeLinearLayer : public Layer
+    {
+    public:
+        static Ptr<DequantizeLinearLayer> create(const LayerParams& params);
+    };
+
+    class CV_EXPORTS QLinearAddLayer : public Layer
```

Comment:
It makes sense to add documentation with links on similar layers/nodes (e.g, from ONNX)